src/design/ButtonBar/index.stories.tsx

```diff
@@ -141,7 +175,7 @@ namespaced which would have a DX impact that is not worth it for now.
         </Button>
         <Tooltip>
           <TooltipTrigger asChild>
-            <div>
+            <div className="!border-0">
```

Comment:
We somehow have to find a way to get rid of this with a tailwind css selector in the ButtonBar Component? Removing the border should all be handled by the ButtonBar. 


I think this also broke this part of our workflow editor

![CleanShot 2023-08-03 at 09 18 40@2x](https://github.com/direktiv/direktiv-ui/assets/121789579/54a33000-0baf-4567-919d-c417a4b48d26)